content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -93,7 +93,10 @@ state file `cpu_manager_state` in the kubelet root directory.
 This policy manages a shared pool of CPUs that initially contains all CPUs in the
 node. The amount of exclusively allocatable CPUs is equal to the total
 number of CPUs in the node minus any CPU reservations by the kubelet `--kube-reserved` or
-`--system-reserved` options. CPUs reserved by these options are taken, in
+`--system-reserved` options. From 1.17, the CPU reservation list can be specified
+explictly by kubelet --reserved-cpus option. The explicit CPU list specified by
```

Comment:
I recommend putting “--reserved-cpus” and other flags inside backticks.

Reply:
thanks and updated.